examples/mysql/client.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const opentelemetry = require('@opentelemetry/core');
+const config = require('./setup');
+
+/**
+ * The trace instance needs to be initialized first, if you want to enable
+ * automatic tracing for built-in plugins (HTTP in this case).
```

Comment:
```suggestion
 * automatic tracing for built-in plugins (HTTP and MySQL in this case).
```

Reply:
In this case this is the client, it is just HTTP.